CHANGELOG.md

```diff
@@ -1,5 +1,84 @@
+    [issue/492]: https://github.com/cloudflare/wrangler/issues/492
+
+  - **Provide helpful error when user accidentally puts kv-namespace config under [site] - [gabbifish], [issue/798] [pull/937]**
+
+    TOML formatting can be tricky, specifically tables, so it is common for users unfamiliar with the format to accidentally nest attributes inside tables without intending it. In this case, if a user adds a kv-namespaces entry to the bottom of a toml with [site] configuration already declared, it is parsed as a part of the [site] table, rather than as a top-level key. The error output from this is not super helpful, as it just says "unknown field `kv-namespaces`" which isn't precisely correct.
+
+    This PR detects when this error occurs and provides a help suggestion to put kv-namespaces ABOVE the [site] table entry to fix the problem.
+
+    [gabbifish]: https://github.com/gabbifish
+    [pull/937]: https://github.com/cloudflare/wrangler/pull/937
+    [issue/798]: https://github.com/cloudflare/wrangler/issues/798
+
+- ### Maintenance
+
+  - **Make Azure use latest `rustc` - [EverlastingBugstopper], [issue/887] [pull/893]**
+
+    Updates our CI to update the rust toolchain to the latest stable version after installation.
+
+    [EverlastingBugstopper]: https://github.com/EverlastingBugstopper
+    [pull/893]: https://github.com/cloudflare/wrangler/pull/893
+    [issue/887]: https://github.com/cloudflare/wrangler/issues/887
+
+  - **Fix nightly builds - [EverlastingBugstopper], [pull/895], [pull/898]**
+
+    We caught a bug in our CI that was causing nightly builds to fail in Azure Pipelines
+
+    [EverlastingBugstopper]: https://github.com/EverlastingBugstopper
+    [pull/895]: https://github.com/cloudflare/wrangler/pull/895
+    [pull/898]: https://github.com/cloudflare/wrangler/pull/898
+
+  - **Fix compiler warnings on windows - [uma0317], [issue/800] [pull/919]**
+
+    We build Wrangler for Mac OSX, Linux, and Windows, and each of these environments has slightly different needs at compile time. In this case, community contributor @uma0317 added configuration that eliminated unused imports for Windows at compile time. Thanks for the PR @uma0317!
+
+    [uma0317]: https://github.com/uma0317
+    [pull/919]: https://github.com/cloudflare/wrangler/pull/919
+    [issue/800]: https://github.com/cloudflare/wrangler/issues/800
+
+  - **Remove deprecated kv-namespace config check - [ashleymichal], [pull/929]**
+
+    Back in 1.1.0, we introduced more robust support for adding KV namespaces to your project; however it was a breaking change for users who were still using our first pass at configuration for this in their toml, so we added a friendly error message telling them how to update their toml. We feel likely all of our users have safely transitioned onto the new syntax since then, and so this PR removes that warning; any lingering use of the old syntax will be met with a serde error instead.
```

Comment:
first sentence here is a bit run-on - also "serde error" should probably be changed since it's an implementation detail that users likely wont understand